rb/lib/selenium/webdriver/remote/bridge.rb

```diff
@@ -568,6 +569,52 @@ def shadow_root(element)
           ShadowRoot.new self, shadow_root_id_from(id)
         end
 
+        #
+        # virtual-authenticator
+        #
+
+        def virtual_authenticator_id
+          @authenticator_id
+        end
+
+        def add_virtual_authenticator(options)
+          @authenticator_id = execute :add_virtual_authenticator, {}, options.to_dict
+        end
+
+        def remove_virtual_authenticator
+          execute :remove_virtual_authenticator, {}, {authenticatorId: @authenticator_id}
+          @authenticator_id = nil
+        end
+
+        def add_credential(credential)
+          credential = credential.to_dict
+          credential[:authenticatorId] = @authenticator_id
+          execute :add_credential, {}, credential
+        end
+
+        def credentials
+          credential_data = execute :get_credentials, {}, {authenticatorId: @authenticator_id}
+          credential_list = []
+
+          credential_data.each do |cred|
+            credential_list.push(Credential.new.from_dict(cred))
+          end
+          credential_list
+        end
+
+        def remove_credential(credential_id)
+          credential_id = Base64.urlsafe_encode64(credential_id.pack('C*')) if credential_id.instance_of?(Array)
+          execute :remove_credential, {}, {credentialId: credential_id, authenticatorId: @authenticator_id}
+        end
+
+        def remove_all_credentials
+          execute :remove_all_credentials, {}, {authenticatorId: @authenticator_id}
+        end
+
+        def user_verified(verified)
```

Comment:
Since this sets user verified, the convention in Ruby would be to name this method with a trailing equals sign:

```suggestion
        def user_verified=(verified)
```

Then the method is called like this:

```ruby
bridge.user_verified = false
```

Reply:
Changes incorporated.